.vscode/settings.json

```diff
@@ -1,6 +1,3 @@
 {
-    "editor.codeActionsOnSave": {
```

Comment:
Ugh, that is annoying...
I'm currently looking at https://github.com/Microsoft/vscode/issues/59912, which if solved, I think would allow us to keep the setting here.

In the meantime, I've been searching for a way to exclude certain files, but it appears I can only disable it for a specific programming language rather than a set of files...

I really want to keep organizing the imports, and hope the issue I linked above might be a good workaround for us. If not, we might simply have to remove this altogether... of which I'm not a big fan...

Reply:
Ok, let's remove it for now, but track of the issue - opened #230 to keep track of this in the future.